packages/site/src/layout/ComponentView.tsx

```diff
@@ -106,6 +106,14 @@ export const ComponentView = () => {
         </>
       ),
     },
+    {
+      label: "Implement",
```

Comment:
This looks okay to me and shouldn't conflict too heavily with either of the link PRs I have in flight.